google/cloud/storage/status_or.h

```diff
@@ -181,8 +185,10 @@ class StatusOr final {
   Status& status() & { return status_; }
   Status const& status() const& { return status_; }
   Status&& status() && { return std::move(status_); }
+#if GOOGLE_CLOUD_CPP_HAVE_CONST_REF_REF
   Status const&& status() const&& { return std::move(status_); }
-  //@}
+#endif  // GOOGLE_CLOUD_CPP_HAVE_CONST_REF_REF
+        //@}
```

Comment:
Is the indentation change significant?  (And below.)

Reply:
No, clang-format just did that, we could insert a blank line or reorder the functions to make it look better.  Frankly I just let clang-format do its thing, most of the time it is good, and I rather worry about other things. I will change it if you think it is important, of course.

Reply:
I'm happy to let clang-format have its way (mostly) too.